iree/test/e2e/regression/generate_e2e_matmul_tests.py

```diff
@@ -115,9 +115,9 @@ def get_test_shapes(shapes_id: ShapesId):
         TestShape(m=10000, k=1, n=1),
         TestShape(m=1, k=10000, n=1),
         TestShape(m=1, k=1, n=10000),
-        #TestShape(m=1, k=1000, n=1000),
-        #TestShape(m=1000, k=1, n=1000),
-        #TestShape(m=1000, k=1000, n=1),
+        TestShape(m=1, k=1000, n=1000),
+        # TestShape(m=1000, k=1, n=1000),
```

Comment:
If commenting out tests then note why (ideally with an issue # of when it should be enabled again) or just remove. Otherwise it looks like someone was doing some local dev, commented them out for some reason, and then accidentally let them slip into the commit. Maybe that happened here :)

Reply:
The diff is a bit misleading: these tests were commented out (with a comment explaining why, above). This PR uncomments some of them. Line 119 is actually a no-op given the `-` line 120 above -- it's just a whitespace change inside a comment.

Reply:
So the actual functional change here is uncommenting some matrix*vector and vector*matrix tests.  The reason is that we now have dedicated kernels for those cases. This uncommenting should have been done in my previous PR adding those kernels, that was just the tests catching up one PR later.

Reply:
cool!